dispatch/theme/fields.py

```diff
@@ -101,6 +103,44 @@ def validate(self, data):
         if not isinstance(data, basestring):
             raise InvalidField('%s data must be a string' % self.label)
 
+class DateTimeField(Field):
+
+    type = 'datetime'
+
+    def validate(self, data):
+        if not parse_datetime(data):
+            raise InvalidField('%s must be valid format' % self.label)
+
+    def prepare_data(self, data):
+        return parse_datetime(data)
+
+class IntField(Field):
+
+    type = 'int'
+
+    def validate(self, data):
+        try:
+            if isinstance(data, int):
+                return int
+            return int(data, base=10)
+        except ValueError:
+            raise InvalidField('%s must be integer' % self.label)
+
+    def prepare_data(self, data):
+        if isinstance(data, int):
+            return data
+        return int(data, base=10)
+
+class UIntField(IntField):
```

Comment:
Instead of using a separate field, could the `IntegerField` accept optional `min_value` and `max_value` keyword args? In my opinion, that would be a cleaner approach, and it's also the way Django configures its built-in db `IntegerField`: https://docs.djangoproject.com/en/1.11/ref/forms/fields/#integerfield